integration-tests/src/test/groovy/uk/gov/justice/digital/hmpps/cattool/DashboardWomenSpecification.groovy

```diff
@@ -0,0 +1,156 @@
+        ]
+      }
+      return JsonOutput.toJson(contents)
+    }
+  }
+
+  static final SECURITY_AUTO = '{"socProfile":{"transferToSecurity":"true"}}'
+
+  /** convenience method to make data setup calls as succinct as possible */
+  def dbRow(bookingId, prisonId, catType, startDate, referredDate, securityReviewedDate, assessmentDate, approvalDate, dueByDate, json, riskProfile = '{}') {
+    db.doCreateCompleteRow(-bookingId, bookingId, json, catType == 'INITIAL' ? 'FEMALE_USER' : 'FEMALE_USER', 'APPROVED', catType, null, referredDate, null, 1, riskProfile,
+      prisonId, "ON${bookingId}", startDate, null, securityReviewedDate, approvalDate, assessmentDate, dueByDate)
+  }
+
+
+  JasonBuilder ic(suggested, overridden = null, supervisor = null) {
+    def b = new JasonBuilder(catType: CatType.INITIAL, suggested: suggested, overridden: overridden, supervisor: supervisor)
+    return b
+  }
+
+
+  def "The initial cat dashboard should show correct women stats data"() {
+    // ignored as AWAITING_APPROVAL or wrong cat type
+    db.doCreateCompleteRow(-10, 700, ic('R').build(), 'FEMALE_USER', 'AWAITING_APPROVAL', 'INITIAL', null, null, null, 1, '{}',
+      'PFI', "ON700", "'2019-07-01Z00:00'", null, null, '2019-07-29', '2019-07-22', '2019-08-03')
+
+    //-------------------------------------------------------------------------------------------------------------
+    // PFI initial
+    dbRow(30, 'PFI', 'INITIAL', "'2019-07-01Z00:00'", null, null, '2019-07-20', '2019-08-01', '2019-08-03', ic('R').build())
+    dbRow(31, 'PFI', 'INITIAL', "'2019-07-01Z00:00'", null, null, '2019-07-21', '2019-07-31', '2019-08-02', ic('R', 'T').build())
+
+    // PFI initial referred to security
+    dbRow(32, 'PFI', 'INITIAL', "'2019-07-01T03:00Z'", "'2019-07-09T15:30Z'", "'2019-07-09T17:40Z'", '2019-07-22', '2019-07-30', '2019-08-03', ic('R').build(), SECURITY_AUTO)
+    dbRow(33, 'PFI', 'INITIAL', "'2019-07-01T04:00Z'", "'2019-07-10T09:00Z'", "'2019-07-11T11:00Z'", '2019-07-23', '2019-07-29', '2019-07-28', ic('T', 'R', 'T').securityType(SecurityType.MANUAL).build())
+    // .. late
+
+    //-------------------------------------------------------------------------------------------------------------
+    // LNI initial
+    dbRow(34, 'LNI', 'INITIAL', "'2019-07-01T00:00Z'", null, null, '2019-07-22', '2019-07-29', '2019-08-03', ic('R', 'T', 'R').build())
+    dbRow(35, 'LNI', 'INITIAL', "'2019-07-01T00:00Z'", null, null, '2019-07-22', '2019-07-29', '2019-08-03', ic('R', 'T').build())
+
+    // LNI initial referred to security
+    dbRow(36, 'LNI', 'INITIAL', "'2019-07-01T00:00Z'", "'2019-07-20T00:00Z'", "'2019-07-29T00:00Z'", '2019-07-22', '2019-07-29', '2019-08-03', ic('R').build(), SECURITY_AUTO)
+    dbRow(37, 'LNI', 'INITIAL', "'2019-07-01T00:00Z'", "'2019-07-20T00:00Z'", "'2019-07-29T00:00Z'", '2019-07-22', '2019-07-29', '2019-08-03', ic('T').securityType(SecurityType.MANUAL).build())
```

Comment:
Additional test would be to create a male prison record here. It shouldn't modify numbers when all female prisons are selected. The opposite can be done in DashboardSpecification.

Reply:
I didnt get this.. we can talk about it